### ISSUE TITLE
Ускорить инициацию браузера с редактором

### ISSUE THREAD

**Issue by tormozit** [salexdv/bsl_console#304]
Сделал форму просмотра текста XML на базе твоего редактора. Каждый раз при открытии этой формы проходит 1+ секунд до вызова платформой события ДокументСформирован у поля HTML с редактором. При частом открытии такой формы задержка до отображения текста становится ощутимой. Можно ли как то ускорить или закэшировать загрузку редактора для сценария частого открытия формы?
Тут https://forum.mista.ru/topic.php?id=884391 пытался кэшировать форму просмотра XML, но похоже платформа пересоздает объект браузера всех полей HTML насильно при открытии формы.

**Comment by salexdv** [comment]
Я пробовал ускорить загрузку, но существенно повлиять на время формирования документа не удалось. 1Ска и так кэширует html, но всё равно грузится довольно долго. В теории можно попробовать собрать отдельную версию редактора, которая включает поддержку только xml. Исходники такой сборки будут занимать меньше места и будут грузиться быстрее, но это увеличит размер твоей сборки т.к. всё таки это отдельный макет с исходниками. Если хочешь, могу попробовать собрать такую версию редактора для пробы или могу подсказать, как ты сам можешь это сделать.

**Comment by tormozit** [comment]
Нет. Отдельная поставка и заметное увеличение размера поставки не стоят того.

**Comment by leobrn** [comment]
Я столкнулся с подобной штукой при внедрении редактора в свою подсистему, при открытии бывает на пару секунд появляется лоудер bsl. @tormozit у Вас такая же проблема?
![Скриншот 06-04-2023 183252](https://user-images.githubusercontent.com/16267179/230366103-3ca54cfe-048e-46ec-aae7-a7e2b736c80f.png)
